panc/src/main/scripts/panlint/panlint.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env python
+# coding=utf8
+#
+# Copyright 2016 quattor.org
```

Comment:
quattor.org isn't a legal entity which could make it difficult to establish the right to future changes. Is QWG a defined entity?